tests/nmstate_varlink/varlink_service_test.py

```diff
@@ -0,0 +1,110 @@
+#
```

Comment:
These seem to be integration tests, then they belong in `tests/integration/varlink_service_test.py`